next.config.js

```diff
@@ -37,7 +37,6 @@ const hideWaypointTipContent = {
 }
 
 module.exports = withHashicorp({
-	nextOptimizedImages: true,
```

Comment:
`nextOptimizedImages` is an outdated approach to image handling that breaks Next's ability to import SVG files. Disabling this does not seem to have any impact, and restores the ability for Next to import SVG files, which is required for our `FlightIcon` component to work correctly.